repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/ProjectRepositoryTest.scala

```diff
@@ -25,26 +25,52 @@ class ProjectRepositoryTest
   }
 
   private val dummyUser: User = TestUserUtils.getDummyUser()
+  private val stranger: User = TestUserUtils.getDummyUser()
   private val dummyProject: Project = TestProjectUtils.getDummyProject(ownerId = dummyUser.userId)
 
   import datastorageModule.{ projectRepository, userRepository }
-  "ProjectRepository" when {
-
-    "getUserById" should {
 
+  "ProjectRepository" when {
+    "getProjectById" should {
       "find newly added project by id" taggedAs Dao in {
         val addUserFuture = userRepository.addUser(dummyUser)
         val result = for {
           _ <- addUserFuture
           _ <- projectRepository.addProject(dummyProject)
           getById <- projectRepository.getProjectById(dummyProject.projectId)
         } yield getById
+
         result.map(optProject => optProject shouldEqual Some(dummyProject))
       }
     }
 
-    "updateProjectVersion" should {
+    "getProjectsByOwnerId" should {
+      "find all user projects" taggedAs Dao in {
+        val addUserFuture = userRepository.addUser(dummyUser)
+        val result = for {
+          _ <- addUserFuture
+          _ <- projectRepository.addProject(dummyProject)
+          getByUserId <- projectRepository.getProjectsByOwnerId(dummyUser.userId)
+        } yield getByUserId
+
+        result.map(optProjects => optProjects shouldEqual List(dummyProject))
+      }
+
+      "not found any project" taggedAs Dao in {
```

Comment:
```suggestion
      "not return projects of another user" taggedAs Dao in {
```

Reply:
I didn`t try to get projects of another User. I just show that we can have projects for a few users but we can get only that project where we are owner.

Reply:
Then give test a better name. Current name implies that repository should not find any project at all (no matter what). It doesn't make sense because we know that repository can and should find projects of the user is owner.

Reply:
You are right. Fixed.